.github/workflows/ci.yml

```diff
@@ -16,7 +16,38 @@ jobs:
     steps:
     - uses: actions/checkout@v3
       with:
-        fetch-depth: 1
+        fetch-depth: 0
+    - name: Get branch name
+      id: branch-name
+      uses: tj-actions/branch-names@v6
```

Comment:
Curious, why do we need this action? Not suggesting we change it, but don't we already have this from the github event payload: `github.ref_name` ?

https://docs.github.com/en/actions/learn-github-actions/contexts